contracts/Aavegotchi/libraries/LibAppStorage.sol

```diff
@@ -364,8 +369,8 @@ struct AppStorage {
     mapping(address => mapping(uint256 => EnumerableSet.UintSet)) depositIdToEquippedGotchis;
```

Comment:
`itemsTokenApprovals` is not part of `Auxilliary structs`, move to `Items Roles Registry` section